### ISSUE TITLE
DLSS Unavailable Despite having Correct DLLs

### ISSUE THREAD

**Issue by JollyDoesItSo** [sultim-t/vkquake-rt#23]
Absolutely blown away by this project however I've only been able to use AMD FSR on the game despite having followed the instructions to install the files for DLSS 2.0. I'm running the game on an RTX 3060 and I also had the same issue with Serious Sam RT, are there particular settings I have to disable for it to be available to be enabled?

Current Settings Menu posted bellow
![20221127133056_1](https://user-images.githubusercontent.com/91629547/204137993-c8c9dcea-768f-44a3-8f14-c0a3caeb976e.jpg)

**Comment by tabnul** [comment]
same here, quake bought from steam, windows 11, 3080ti. correct dlls but nu dlss option.

**Comment by WabbitToonz** [comment]
is  "nvngx_dlss.dll"  file  in the same folder  as  "vkQuake.exe"?
https://github.com/sultim-t/vkquake-rt/releases/tag/1.20.2-rt1.0.1  has  "Installing DLSS 2.0

**Comment by JollyDoesItSo** [comment]
> 

Yes I've got both of them located in the root folder where the vkQuake.exe is located and still is unavailable

**Comment by Berzerk2k2** [comment]
Try using an older nvngx_dlss.dll like **2.2.6** from [Tech PowerUp - NVIDIA DLSS DLL](https://www.techpowerup.com/download/nvidia-dlss-dll/) (recommended by Digital Foundry in their [Quake RTX video](https://www.youtube.com/watch?v=IhhLcZQ2zD0)). I was not able to get NVIDIA DLSS working with the dll that is attached in the release section but with 2.2.6 it worked just fine. Same goes for **Serious Sam RTX**. I guess the attached dll file from the release section is simply broken.

**Comment by JollyDoesItSo** [comment]
> Reference in new

BINGO! This did it Berzerk! Swapping to this DLSS DLL made DLSS 2 work properly and I'm getting some REALLY nice frames with it

**Comment by NebularNerd** [comment]
> [@SkeleTonHammer](https://www.youtube.com/channel/UC2o3cHwnGPHF8bsZkD-oiqg)
> [2 days ago (edited)](https://www.youtube.com/watch?v=IhhLcZQ2zD0&lc=UgxTwB8UDYUEi36XfOJ4AaABAg)
> You say later versions of DLSS introduce ghosting - this is half true. Some later versions yes, however there is at least one VERY recent version of DLSS that does not have this ghosting issue. The one I've personally tested is 2.4.12 (there's a v2 version of 2.4.12, do not use v1 as that has ghosting issues - the v2 is sourced from Plague Tale: Requiem).

This was from the YT comments of Digital Foundry's video. Can confirm 2.4.12 v2 seems to works ok as well.

**Comment by r3538987** [comment]
Can this be hardware related? 
Running on RTX2070S, and I have same issue as OP, only FSR is available on mine setup.
Different DLSS dll files doesn't change anything, even with DF video mentioned one - v2.2.6.

**Comment by NebularNerd** [comment]
> Can this be hardware related? Running on RTX2070S, and I have same issue as OP, only FSR is available on mine setup. Different DLSS dll files doesn't change anything, even with DF video mentioned one - v2.2.6.
> 
> Whats is the difference between **_RayTracedGL1-DLSS_** and **_RayTracedGL1_**? Tried both in combo with different **nvngx_dlss** files, but no luck.

**_RayTracedGL1-DLSS_** is for NVidia cards, you should just need to download it, extract over the the existing RayTracedGL1.dll, add your nvngx_dlss.dll of choice and be good to go. It should be OK on a 2070 as it runs a treat on my Zotac 2060 Super. In video options you should then be able to turn off `AMD FSR 2.0` and set `NVIDIA DLSS` to your preference.


![vkQuake 1 20 3 31_12_2022 09_15_07](https://user-images.githubusercontent.com/8470449/210131634-0826ab2c-5718-4619-8602-41cdf1307836.png)

**Comment by r3538987** [comment]
Thanks for explaining those DLLs, but still on my machine DLSS does not seems to appear to load. 
Send exactly same archive to friend with RTX3060, and it worked out of the box.

**Comment by donizettilorenzo** [comment]
Me too.

Please update the package to be compatible with lastest drivers and DLSS 3.x DLL
